### DISCUSSION TITLE
Allow delivery customizations with Shop Pay

### DISCUSSION THREAD

**Discussion by nboliver** [Shopify/function-examples#219] [Feedback: Delivery Customization API]
If there is no way to use the delivery customization API with Shop Pay, it limits use quite significantly. For example, there may be situations where a free / cheaper shipping method should be removed based on the contents of the cart - if that method isn't removed, customers using Shop Pay will get free shipping when they aren't supposed to.

**Comment by gil--** [comment]
Not sure if you participated in the Discord AMA with functions team last week but this was asked and they mentioned Shop pay support coming in the next couple months: https://discord.com/channels/842813079926603828/1096089706330062858/1096091110604021800

**Reply by nboliver** [reply]
Thanks @gil-- ! That's great news. Will dig through the rest of the info from the AMA to get caught up.
